bin/airgap-setup-parsing.sh

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env bash
+
+source bin/common.sh
+
+TMP_DIR=$(mktemp -d -t sas.mon.XXXXXXXX)
+
+required_vars=("AIRGAP_REGISTRY" "CR_USERNAME" "CR_PASSWORD")
+for var in "${required_vars[@]}"; do
+    if [ -z "${!var}" ]; then
+        log_error "ERROR: Environment variable $var is not set."
```

Comment:
Doesn't this result in a error message with two mentions of ERROR, e.g. ERROR: ERROR: Environment variable..."?  The `log_error`  (and the other log_* functions) prefix the log message with the level/severity indicator.